OWNERS.md

```diff
@@ -0,0 +1,14 @@
+* See [CONTRIBUTING.md](CONTRIBUTING.md) for general contribution guidelines.
+
+This page lists all active maintainers and their areas of expertise. This can be used for
+routing PRs, questions, etc. to the right place.
+
+# Maintainers
+
+* Otto van der Schaaf ([oschaaf](https://github.com/oschaaf)) (oschaaf@we-amp.com)
+  * All things Nighthawk
+* Harvey Tuch ([htuch](https://github.com/htuch)) (htuch@google.com)
+  * Envoy internals, architecture and performance methodology.
+* John Plevyak ([jplevyak](https://github.com/jplevyak)) (jplevyak@google.com)
+  * Envoy internals and architecture.
+* Jakub Sobon ([mum4k](https://github.com/mum4k)) (???@google.com)
```

Comment:
The email is: mumak@google.com

For now you can list me on:
  *  APIs, general functionality and miscellany.